doc/source/whatsnew/v0.18.2.txt

```diff
@@ -19,6 +19,69 @@ Highlights include:
 New features
 ~~~~~~~~~~~~
 
+.. _whatsnew_0182.enhancements.asof_merge:
+
+``pd.merge_asof()`` for asof-style time-series joining
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A long-time requested feature has been added thru the ``pd.merge_asof()`` function, to
```

Comment:
Maybe ``pd.merge_asof()`` -> ``` :func:`merge_asof` ``` (I know we don't do that regularly in the whatsnew files, but we maybe should, and it is nice people can click on that)